cmd/drone-hugo/main.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/drone-plugins/drone-hugo"
+	"github.com/urfave/cli"
+)
+
+var build = "0" // build number set at compile-time
+var version = "2.0"
```

Comment:
Maybe you could use the same version scheme as the other plugins? This will be set automatically based on the git tag.